gitlab-runner-17.2.yaml

```diff
@@ -45,27 +45,36 @@ subpackages:
           output: gitlab-runner-helper
           ldflags: -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
-  - name: "gitlab-runner-compat-${{vars.short-package-version}}"
-    description: "Gitlab-runner Compat package for alpine"
+  - name: "gitlab-runner-oci-entrypoint-${{vars.short-package-version}}"
+    description: "Gitlab-runner oci entrypoint for alpine"
     dependencies:
       provides:
-        - gitlab-runner-compat=${{package.full-version}}
+        - gitlab-runner-oci-entrypoint=${{package.full-version}}
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"
           cp dockerfiles/runner/alpine/entrypoint "${{targets.subpkgdir}}"/entrypoint
           chmod 755 "${{targets.subpkgdir}}"/entrypoint
 
+  - name: "gitlab-runner-helper-oci-entrypoint-${{vars.short-package-version}}"
+    description: "Gitlab-runner-helper oci entrypoint for alpine"
+    dependencies:
+      provides:
+        - gitlab-runner-helper-oci-entrypoint=${{package.full-version}}
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.subpkgdir}}"/usr/bin
+          cp dockerfiles/runner-helper/scripts/gitlab-runner-build "${{targets.subpkgdir}}"/usr/bin/gitlab-runner-build
+          cp dockerfiles/runner-helper/helpers/entrypoint "${{targets.subpkgdir}}"/entrypoint
+
   - name: "gitlab-runner-helper-compat-${{vars.short-package-version}}"
-    description: "Gitlab-runner-helper Compat package for alpine"
+    description: "Gitlab-runner-helper compat for alpine"
     dependencies:
       provides:
         - gitlab-runner-helper-compat=${{package.full-version}}
     pipeline:
       - runs: |
           mkdir -p "${{targets.subpkgdir}}"/usr/bin
-          cp dockerfiles/runner-helper/scripts/gitlab-runner-build "${{targets.subpkgdir}}"/usr/bin/gitlab-runner-build
-          ls -alh packaging/root/usr/bin
           ln -sf /usr/bin/gitlab-runner "${{targets.subpkgdir}}"/usr/bin/gitlab-ci-multi-runner
 
```

Comment:
Theres also no melange tests in here - can we try and add some? is there a --help or a --version or something we can invoke? can we validate the symlinks when adding the entrypoint package etc?